apps/web/src/pages/templates/editor/TemplateEditorPage.tsx

```diff
@@ -12,9 +12,10 @@ import { TemplateTriggerModal } from '../../../components/templates/TemplateTrig
 import { TemplateInAppEditor } from '../../../components/templates/in-app-editor/TemplateInAppEditor';
 import { TriggerSnippetTabs } from '../../../components/templates/TriggerSnippetTabs';
 import { AddChannelsPage } from './AddChannelsPage';
-import { Button } from '../../../design-system';
+import { Button, Switch } from '../../../design-system';
 import { EmailMessagesCards } from '../../../components/templates/email-editor/EmailMessagesCards';
 import { TemplateSMSEditor } from '../../../components/templates/TemplateSMSEditor';
+import { useStatusChangeControllerHook } from '../../../legacy/pages/templates/editor/use-status-change-controller.hook';
```

Comment:
Can we move this file and any other dependents file out of the legacy folder? I want to remove this folder once we finished moving all pages.